src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/PlaygroundResultFeatureStrategyList/PlaygroundResultStrategyList/PlaygroundResultFeatureStrategyItem/PlaygroundResultStrategyExecution/PlaygroundResultConstraintExecution/PlaygroundResultConstraintAccordion/PlaygroundResultConstraintAccordionView/PlaygroundConstraintAccordionViewHeader/PlaygroundConstraintAccordionViewHeaderInfo/PlaygroundContraintAccordionViewHeaderMultipleValues/PLaygroundConstraintAccordionViewHeaderMultipleValues.tsx

```diff
@@ -0,0 +1,85 @@
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { styled, Typography } from '@mui/material';
+import React, { useEffect, useMemo, useState } from 'react';
+import classnames from 'classnames';
+import { useStyles } from '../../../PlaygroundConstraintAccordion.styles';
+import { PlaygroundConstraintSchema } from 'hooks/api/actions/usePlayground/playground.model';
+
+const StyledValuesSpan = styled('span')(({ theme }) => ({
+    display: '-webkit-box',
+    WebkitLineClamp: 2,
+    WebkitBoxOrient: 'vertical',
+    overflow: 'hidden',
+    wordBreak: 'break-word',
+    fontSize: theme.fontSizes.smallBody,
+    margin: 'auto 0',
+    [theme.breakpoints.down(710)]: {
+        margin: theme.spacing(1, 0),
+        textAlign: 'center',
+    },
+}));
+
+interface ConstraintSingleValueProps {
```

Comment:
Should this be renamed? Looks like a leftover from a copy, but I might be wrong 🤔 

```suggestion
interface ConstraintMultiValueProps {
```